src/utils/resources.js

```diff
@@ -29,7 +30,7 @@ import { isNil } from 'lodash'
 export const generateUri = (item, tab) => {
   let uri = `store://${tab}/${item.project}/`
 
-  if (tab === MODELS_TAB || tab === DATASETS_TAB || tab === FILES_TAB) {
+  if (tab === MODELS_TAB || tab === DATASETS_TAB || tab === FILES_TAB || tab === ARTIFACTS_TAB) {
```

Comment:
you can delete `FILES_TAB` from this condition. 'files' isn't a valid value in this function